WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -615,6 +621,10 @@ private extension ShippingLabelAddressFormViewController {
                                                             comment: "Error in finding the address in the Shipping Label Address Validation in Apple Maps")
         static let phoneNumberErrorNotice = NSLocalizedString("The phone number is not valid or you can't call the customer from this device.",
             comment: "Error in calling the phone number of the customer in the Shipping Label Address Validation")
+        static let addressValidationErrorNotice = NSLocalizedString("Certain required fields needs attention.",
+                comment: "Ship From address update failed notice")
+        static let addressValidationFailedNotice = NSLocalizedString("The address cannot be validated, please double-check the address and try again",
+                comment: "Ship From address validation failed notice")
```

Comment:
Similar comments as above:
```suggestion
        static let addressRemoteValidationFailedNotice = NSLocalizedString("The address cannot be validated, please double-check it and try again",
                comment: "Generic error message when remote validation fails in Shipping Label Address Validation")
```